math/libfaiss/Portfile

```diff
@@ -1,62 +1,77 @@
+                    evaluation and parameter tuning. Faiss is written\
+                    in C++ with complete wrappers for\
+                    Python/numpy. Some of the most useful algorithms\
+                    are implemented on the GPU. It is developed by\
+                    Facebook AI Research.
+
+checksums           rmd160  3b1180dc7ceadc484bb86cfe32fb5cd1c4a694b9 \
+                    sha256  6962641a9183e12d394aa2f9c443593734a373d60c61fd0633edda5029d2fa78 \
+                    size    2762193
+
+depends_lib-append  port:libomp
+
+post-extract {
+    # fix destroot commands like `$(PYTHON) setup.py install`
+    reinplace -E \
+        "s|^(\[\[:space:]]+\\\$\\(PYTHON\\)\[\[:space:]]+setup\.py\[\[:space:]]+)(install)|\\1 --no-user-cfg \\2 ${destroot.destdir}|" \
+        ${worksrcpath}/python/Makefile
+}
 
 compilers.choose    cxx
-# The Apple native llvm compiler does not support OpenMP.
+# The Apple native llvm compiler does not support OpenMP
 compiler.blacklist  llvm-gcc-4.2
-compiler.fallback   macports-gcc-7
+compiler.fallback   macports-gcc-9
 compilers.setup
 
-use_configure       no
+configure.args-append \
+                    --without-cuda
+configure.cppflags-append \
+                    -I${prefix}/include/libomp
+configure.cflags-append \
+                    -fPIC -m64 -Wall -g -O3 -msse4 -mpopcnt -fopenmp \
+                    -Wno-sign-compare -std=c++11
+configure.ldflags-append \
+                    -g -fPIC -fopenmp -L${prefix}/lib/libomp
+configure.cxx_stdlib libstdc++
 
-pre-build {
-    touch ${worksrcpath}/makefile.inc
-}
-build.env-append    CC=${configure.cxx}
-build.env-append    "CFLAGS=-fPIC -m64 -Wall -g -O3 -msse4 -mpopcnt -fopenmp \
-                            -Wno-sign-compare -std=c++11 -I/usr/include/malloc/"
-build.env-append    "LDFLAGS=-g -fPIC -fopenmp"
-build.env-append    SHAREDEXT=dylib
-build.env-append    "SHAREDFLAGS=-Wl,-F. -bundle -undefined dynamic_lookup"
-build.env-append    FAISSSHAREDFLAGS=-dynamiclib
-build.env-append    BLASCFLAGS=-DFINTEGER=int
-build.env-append    "BLASLDFLAGS=-framework Accelerate"
-build.target        libfaiss.a libfaiss.dylib
+build.env-append    SHAREDEXT=dylib \
+                    "SHAREDFLAGS=-Wl,-F. -bundle -undefined dynamic_lookup" \
+                    FAISSSHAREDFLAGS=-dynamiclib \
+                    BLASCFLAGS=-DFINTEGER=int \
+                    "BLASLDFLAGS=-framework Accelerate"
 
-destroot.target
-destroot.destdir
-destroot.cmd        mkdir -p ${destroot}${prefix}/lib && \
-                    cp libfaiss.a libfaiss.dylib ${destroot}${prefix}/lib && \
-                    mkdir -p ${destroot}${prefix}/include/faiss && \
-                    cp -R ./*.h ${destroot}${prefix}/include/faiss/
+build.target        libfaiss.a libfaiss.dylib
 
 variant openblas description {Use OpenBLAS instead of Apple's Accelerate framework} {
-    depends_lib-append  port:OpenBLAS
+    depends_lib-append  port:openblas
```

Comment:
the port name is ```OpenBLAS``` so leave this as it was

Reply:
Thanks. Superseded by https://github.com/macports/macports-ports/pull/7514 with your suggestions.